### DISCUSSION TITLE
Background audio noise in v2.33

### DISCUSSION THREAD

**Discussion by gruedisueli** [ElectroTechnique/TSynth-Teensy4.1#120] [T4.1 code development]
When updating to v2.33 I note that the TSynth has a constant background buzz on the audio line, regardless of patch or volume. I can confirm that this is not the case in v2.32 or 2.31. Here's an audio sample, you should wear headphones to hear it because it's somewhat quiet.
[Tsynth noise on v2.33.zip](https://github.com/ElectroTechnique/TSynth-Teensy4.1/files/8837904/Tsynth.noise.on.v2.33.zip)

**Comment by Electrotechnique** [comment]
Probably because I increased the SPI speed to update the screen faster. Perhaps I'll drop this back. Noise on TSynth is either the SPI clock or from USB connections or ground loops.

**Comment by nekton2** [comment]
I also have a lot of noise like @gruedisueli despite using a power bank. Is there a solution now? Otherwise the TSynth is unusable.

**Reply by Electrotechnique** [reply]
Is this noise from USB or coming from the SPI display? You could attach a sound file as I know what all the various noise sources are now.

**Reply by nekton2** [reply]
The noise is louder when a headphones is plugged directly into the TSynth. A line out level is much lower? Because it's a headphone output? And not a line out, which is not use on the audio board ?

The noise is about 2-3 times louder in v2.33 than in V2.32.

How can I store a file here?

**Reply by Electrotechnique** [reply]
You can attach files by pressing the paperclip.

**Reply by nekton2** [reply]
I have a ZIP file with the audio and get the error "We don’t support that file type.". Sorry, no idea why.

**Reply by Electrotechnique** [reply]
[Test.zip](https://github.com/ElectroTechnique/TSynth-Teensy4.1/files/13399293/Test.zip)
Test

**Reply by Electrotechnique** [reply]
Zip files will upload.

**Reply by nekton2** [reply]
not here :-(

![Screenshot](https://github.com/ElectroTechnique/TSynth-Teensy4.1/assets/130217816/c140bef5-06b2-4cc8-aa26-80d190ab77c8)

**Comment by nekton2** [comment]
v2.33
The noise may change when you turn the encoder and the display changes.

https://github.com/ElectroTechnique/TSynth-Teensy4.1/assets/130217816/218ae15f-d330-4a3a-8fbf-08146ef22292

**Reply by Electrotechnique** [reply]
It's SPI noise from the display. I'll compile another firmware with the SPI speed lower and you can see if that makes a difference.

**Comment by Electrotechnique** [comment]
[TSynthV2_35s.zip](https://github.com/ElectroTechnique/TSynth-Teensy4.1/files/13412104/TSynthV2_35s.zip)

Try this please.

**Reply by nekton2** [reply]
Okay, the noise hasn't gone away, but it's noticeably quieter!
